MultiVersions/Forge/main/integrateddynamics/lang/zh_tw.json

```
@@ -0,0 +1,1873 @@
+    "operator.integrateddynamics.itemstack.size.info": "物品當前的堆疊數量",
+    "operator.integrateddynamics.itemstack.maxsize": "最大堆疊數量",
+    "operator.integrateddynamics.itemstack.maxsize.info": "物品最大的堆疊數量",
+    "operator.integrateddynamics.itemstack.stackable": "是否可堆疊",
+    "operator.integrateddynamics.itemstack.stackable.info": "檢查物品是否至少能堆疊兩個",
+    "operator.integrateddynamics.itemstack.damageable": "是否會損壞",
+    "operator.integrateddynamics.itemstack.damageable.info": "檢查物品是否會損壞",
+    "operator.integrateddynamics.itemstack.damage": "損壞度",
+    "operator.integrateddynamics.itemstack.damage.info": "物品當前的損壞度",
+    "operator.integrateddynamics.itemstack.maxdamage": "最大損壞度",
+    "operator.integrateddynamics.itemstack.maxdamage.info": "物品最大的損壞度",
+    "operator.integrateddynamics.itemstack.enchanted": "是否附魔",
+    "operator.integrateddynamics.itemstack.enchanted.info": "檢查物品是否有附魔",
+    "operator.integrateddynamics.itemstack.enchantable": "是否可附魔",
+    "operator.integrateddynamics.itemstack.enchantable.info": "檢查物品是否能被附魔",
+    "operator.integrateddynamics.itemstack.repaircost": "修理費用",
+    "operator.integrateddynamics.itemstack.repaircost.info": "物品修理的經驗等級",
+    "operator.integrateddynamics.itemstack.rarity": "稀有度",
+    "operator.integrateddynamics.itemstack.rarity.info": "物品的稀有度",
+    "operator.integrateddynamics.itemstack.strength": "物品強度",
+    "operator.integrateddynamics.itemstack.strength.info": "給定物品對上特定方塊的強度",
+    "operator.integrateddynamics.itemstack.canharvest": "是否可挖掘",
+    "operator.integrateddynamics.itemstack.canharvest.info": "檢查給定的方塊是否可以被給定的物品挖掘",
+    "operator.integrateddynamics.itemstack.block": "物品轉為方塊",
+    "operator.integrateddynamics.itemstack.block.info": "取得物品的方塊形式",
+    "operator.integrateddynamics.itemstack.isfluidstack": "是否含有液體",
+    "operator.integrateddynamics.itemstack.isfluidstack.info": "檢查物品是否裝有液體",
+    "operator.integrateddynamics.itemstack.fluidstack": "內含液體",
+    "operator.integrateddynamics.itemstack.fluidstack.info": "物品內含的液體",
+    "operator.integrateddynamics.itemstack.fluidstackcapacity": "液體容量",
+    "operator.integrateddynamics.itemstack.fluidstackcapacity.info": "物品的液體容量（mB）",
+    "operator.integrateddynamics.itemstack.isnbtequal": "NBT 是否相等",
+    "operator.integrateddynamics.itemstack.isnbtequal.info": "檢查兩項物品的 NBT 是否相同",
+    "operator.integrateddynamics.itemstack.isitemequalnonbt": "物品是否相同（不含 NBT）",
+    "operator.integrateddynamics.itemstack.isitemequalnonbt.info": "檢查兩項物品堆疊是否相同（忽略 NBT ，但會檢查損壞度）",
+    "operator.integrateddynamics.itemstack.israwitemequal": "物品是否相同（純物品）",
+    "operator.integrateddynamics.itemstack.israwitemequal.info": "檢查兩項物品堆疊是否相同（忽略 NBT 和損壞度）",
+    "operator.integrateddynamics.itemstack.mod": "來源模組",
+    "operator.integrateddynamics.itemstack.mod.info": "物品來源的模組名字",
+    "operator.integrateddynamics.itemstack.burntime": "燃燒時長",
+    "operator.integrateddynamics.itemstack.burntime.info": "物品作為燃料的燃燒時長（tick）",
```

參考原版將**tick**譯作**刻**
```suggestion
    "operator.integrateddynamics.itemstack.burntime.info": "物品作為燃料的燃燒時長（刻）",
```